dotcom-rendering/src/web/components/FixedMediumFastXI.tsx

```diff
@@ -0,0 +1,103 @@
+import type { DCRContainerPalette } from '../../types/front';
+import type { TrailType } from '../../types/trails';
+import { Card100, Card50_Card25_Card25, Card66_Card33 } from '../lib/slices';
+import { LI } from './Card/components/LI';
+import { UL } from './Card/components/UL';
+import { FrontCard } from './FrontCard';
+
+type Props = {
+	trails: TrailType[];
+	containerPalette?: DCRContainerPalette;
+	showAge?: boolean;
+};
+
+export const FixedMediumFastXI = ({
+	trails,
+	containerPalette,
+	showAge,
+}: Props) => {
+	switch (trails.length) {
+		case 0: {
+			return null;
+		}
+		case 1: {
+			return (
+				<Card100
+					cards={trails}
+					containerPalette={containerPalette}
+					showAge={showAge}
+				/>
+			);
+		}
+		case 2: {
+			return (
+				<Card66_Card33
+					cards={trails}
+					containerPalette={containerPalette}
+					showAge={showAge}
+				/>
+			);
+		}
+		case 3: {
+			return (
+				<Card50_Card25_Card25
+					cards={trails}
+					containerPalette={containerPalette}
+					showAge={showAge}
+				/>
+			);
+		}
+		case 4:
+		case 5:
+		case 6:
+		case 7:
+		case 8:
+		case 9:
+		default: {
```

Comment:
What’s the benefit of having 5-9 explicitely laid out?

```suggestion
		case 4: 
		default: {
```

Reply:
Makes sense 👍